modules/notification/webhook/error.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
Why not put them under `modules/webhook/`?

Reply:
Because that directory already existed.
But okay, I can also move it to a separate directory.

Reply:
Done